### PR TITLE
Created a new entry for numpy.aMax

### PR DESCRIPTION
**Pull request by mark-nettleton**
### Description

Added new entry for numpy function AMax

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- `a`: (Input) The array of elements to choose the maximum from.
+- `axis`: (Default = None) An int or a tuple of ints specifying the axis/axes along which to operate. If a tuple of ints, the maximum is across multiple axes.
+- `out`: (Default = None) An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as the expected output.
+- `keepdims`: (Default = <no value>) A boolean; if `True` will keep reduced axes in the result as dimensions with size one.
+- `inital`: (Default = <no value>) The minimum value of an output element.
+- `where`: (Default = <no value>) A boolean array that maps to `a` with identical dimensions and selects which elements to compare for the maximum.
+
+Returns:
+The maximum element in the array `a`.
+- If axis is None, the result would be a scalar value.
```
Comment (SSwiniarski) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:38]:
```suggestion
- If `axis` is None, the result would be a scalar value.

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- If axis is an integer, the result would be an array of dimension `a.ndim - 1`.
```
Comment (SSwiniarski) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:39]:
```suggestion
- If `axis` is an integer, the result would be an array of dimension `a.ndim - 1`.

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- If axis is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.
```
Comment (SSwiniarski) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:40]:
```suggestion
- If `axis` is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+print(np.amax(nd))
+print(np.amax(nd, axis=0))
+print(np.amax(nd, axis=1))
+```
+
+This produces the following output:
+
+```shell
+16
+[10,12,14,16]
```
Comment (SSwiniarski) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:60]:
```suggestion
[10 12 14 16]
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@mark-nettleton Looks good, only a few minor comments on first review.

**Comment by mark-nettleton** [comment]
@SSwiniarski thanks for the suggestions. All committed. 
I'm still learning github...how do I update my local aMax branch with the changes made with these 4 commits? Do I need to wait until you've merged the branch in to main and then sync with my version of the repo?

**Review by SSwiniarski — APPROVED** [review]
@mark-nettleton Looks good for a second review.

**Comment by mark-nettleton** [comment]
> > @SSwiniarski thanks for the suggestions. All committed. I'm still learning github...how do I update my local aMax branch with the changes made with these 4 commits? Do I need to wait until you've merged the branch in to main and then sync with my version of the repo?
> 
> @mark-nettleton you should be able to do a `git pull` from the remote version of your branch.

@SSwiniarski I get the error: "Your configuration specifies to merge with the ref 'refs/heads/amax'
from the remote, but no such ref was fetched."
Have tried deleting the local branch and recereating from remote repo but same error exists (although it has now updated the code file now). Why is that error occuring?

**Comment by SSwiniarski** [comment]
@KTom101, @yangc95, @caupolicandiaz any idea on @mark-nettleton's issue?

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+---
+Title: '.amax()'
+Description: 'Returns the maximum of an array or maximum along an axis.'
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:3]:
```suggestion
Description: 'Returns the maximum of a given array or maximum along an axis.'

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`.amax()`** function returns the maximum value of an array or maximum along an axis. The `.amax()` function is equivalent to the NumPy method `ndarray.max()`.
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:17]:
```suggestion
The **`.amax()`** function returns the maximum value of a given array or the maximum along an axis. The `.amax()` function is equivalent to the NumPy method `ndarray.max()`.

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+Parameters of the `.amax()` function:
+
+- `a`: (Input) The array of elements to choose the maximum from.
+- `axis`: (Default = None) An int or a tuple of ints specifying the axis/axes along which to operate. If a tuple of ints, the maximum is across multiple axes.
+- `out`: (Default = None) An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as the expected output.
+- `keepdims`: (Default = <no value>) A boolean; if `True` will keep reduced axes in the result as dimensions with size one.
+- `inital`: (Default = <no value>) The minimum value of an output element.
+- `where`: (Default = <no value>) A boolean array that maps to `a` with identical dimensions and selects which elements to compare for the maximum.
+
+Returns:
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:36]: add an empty line after Returns:

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+The maximum element in the array `a`.
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:37]:
```suggestion

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+- If `axis` is None, the result would be a scalar value.
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:38]: remove the empty lines between these items

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+
+- If `axis` is an integer, the result would be an array of dimension `a.ndim - 1`.
+
+- If `axis` is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.
+
+
+## Example
+
+The following example creates an array and then uses a few `.amin()` operations to return the minimum.
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:47]:
```suggestion
The following example creates an array and applies several `.amin()` operations, and returns each result to console.

```

File: content/numpy/concepts/built-in-functions/terms/amax/amax.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+```shell
+16
+[10 12 14 16]
+[ 8 16]
+```
+
+## Codebyte Example
+
+The following example creates a 2-dimensional array each containing 8 random integers (between 1 and 20). First, it prints the array, then performs four `.amax()` operations with different arguments to return the maximum values.
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/amax/amax.md:69]:
```suggestion
The following example creates a 2-dimensional array composed of random integers (between 1 and 20). The code prints the array, and then prints the result of four `.amax()` operations that use different arguments.

```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@mark-nettleton I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by mark-nettleton** [comment]
@caupolicandiaz I've approved your suggestions where I can. However I can't make any further amendments to the code (removing and addign blank lines) due to my issue:
_Your configuration specifies to merge with the ref 'refs/heads/amax'
from the remote, but no such ref was fetched._

**Review by caupolicandiaz — APPROVED** [review]
@mark-nettleton I've gone ahead and made the edits. For future reference, there should never be multiple lines between elements in an entry.

**Comment by SSwiniarski** [comment]
@mark-nettleton Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/numpy/built-in-functions/amax

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**